common/megastructures/gatewayBuild_foreign.txt

```diff
@@ -257,6 +257,8 @@ gatewayBuild_gateway_new = {
 			remove_megastructure = this
 		}
 		if = {
+			# Vanilla code. Needed?
```

Comment:
Yeah, I have also thought about this change in vanilla.
It will not harm but I think in our case it is not needed since we do not have any events that trigger our special gateway building. Whatever, we can put it in.